code/controllers/subsystems/nano.dm

```diff
@@ -1,42 +1,10 @@
-SUBSYSTEM_DEF(nano)
+PROCESSING_SUBSYSTEM_DEF(nano)
```

Comment:
Why all that file not in `processing` subfolder?

Reply:
fixed